### PR TITLE
Rename watch pod to watch bundle

### PR DESCRIPTION
**Pull request by shawn-hurley**
This will rename the watch pod to watch bundle. 

* removed an interface param and brought into line with using the kubernetes client.

### CODE REVIEW

**Review by rthallisey — APPROVED** [review]

**Review by djzager — APPROVED** [review]
ACK

File: bundle/bind.go [hunk 1]
```diff
@@ -53,16 +52,9 @@ func (e *executor) Bind(
 			e.actionFinishedWithError(err)
 			return
 		}
-		k8scli, err := clients.Kubernetes()
```
Comment (eriknelson) [inline bundle/bind.go:56]: :+1:

File: runtime/runtime.go [hunk 2]
```diff
@@ -54,6 +58,7 @@ type Runtime interface {
 	CreateSandbox(string, string, []string, string) (string, error)
 	DestroySandbox(string, string, []string, string, bool, bool)
 	ExtractedCredential
+	WatchRunningBundle(string, string, UpdateDescriptionFn) error
```
Comment (eriknelson) [inline runtime/runtime.go:61]: This looks oddly JS-like :smiling_imp:

File: runtime/watch_bundle.go [hunk 3]
```diff
@@ -54,9 +54,17 @@ func IsErrorCustomMsg(err error) bool {
 	return ok
 }
 
-// WatchPod - watches the pod until completion and will update the last
+// WatchRunningBundleFunc - watches the pod until completion and will update the last
 // description using the UpdateDescriptionFunction
-func WatchPod(podName string, namespace string, podClient v1.PodInterface, updateFunc UpdateDescriptionFn) error {
+type WatchRunningBundleFunc func(string, string, UpdateDescriptionFn) error
+
+func defaultWatchRunningBundle(podName string, namespace string, updateFunc UpdateDescriptionFn) error {
```
Comment (eriknelson) [inline runtime/watch_bundle.go:61]: LGTM

File: runtime/watch_pod_test.go [hunk 4]
```diff
@@ -5,15 +5,19 @@ import (
...
+	"github.com/automationbroker/bundle-lib/clients"
 	core1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/kubernetes/fake"
-	"k8s.io/client-go/kubernetes/typed/core/v1"
 	ktesting "k8s.io/client-go/testing"
 )
 
 func TestWatchPod(t *testing.T) {
```
Comment (eriknelson) [inline runtime/watch_pod_test.go:16]: :tada:

File: runtime/watch_pod_test.go [hunk 5]
```diff
@@ -159,12 +163,13 @@ func TestWatchPod(t *testing.T) {
...
-			podClient, podWatch := tc.PodClient()
+			clientset, podWatch := tc.PodClient()
 			descriptions := []string{}
 			done := make(chan bool)
+			k8scli.Client = clientset
 
 			go func() {
-				watchErr = WatchPod("test", "test", podClient, func(d string) {
+				watchErr = defaultWatchRunningBundle("test", "test", func(d string) {
 					fmt.Printf("NSK: got d -> %v\n", d)
```
Comment (eriknelson) [inline runtime/watch_pod_test.go:173]: You can clip this if you're feeling generous. I can tell its my cruft thanks to my tag :P

**Review by eriknelson — APPROVED** [review]
Looks great! Just one thing I noticed that's actually my fault :)

**Review by jmrodri — APPROVED** [review]
ack

**Pull request merged**